.prow.yaml

```diff
@@ -23,7 +23,7 @@ presubmits:
       timeout: 30m
     spec:
       containers:
-        - image: gcr.io/cloud-foundation-cicd/cft/developer-tools-krm:1.1.3
+        - image: gcr.io/cloud-foundation-cicd/cft/developer-tools-krm:1.3.6
```

Comment:
We probably shouldn't pin to a patch version unless/until we have automation to update this automatically.